contentcuration/contentcuration/utils/assessment/qti/html/flow.py

```
@@ -0,0 +1,55 @@
+
+class HTMLFlowContainer(BlockContentElement):
+    """
+    Base class for HTML elements that can contain flow content
+    (block-level and inline elements).
+    Corresponds to HTML "Flow Content" category.
+    """
+
+    children: FlowGroupList = Field(default_factory=list)
+
+
+class Blockquote(HTMLFlowContainer):
```

Pylint (3.3.7)

[refactor] Too many ancestors (8/7)

(R0901)